components/post_view/post_list_row/post_list_row.tsx

```diff
@@ -16,42 +15,43 @@ import ChannelIntroMessage from 'components/post_view/channel_intro_message/';
 import {isIdNotPost} from 'utils/post_utils';
 import {PostListRowListIds, Locations} from 'utils/constants';
 
-export default class PostListRow extends React.PureComponent {
-    static propTypes = {
-        listId: PropTypes.string.isRequired,
-        previousListId: PropTypes.string,
-        fullWidth: PropTypes.bool,
-        shouldHighlight: PropTypes.bool,
-        loadOlderPosts: PropTypes.func,
-        loadNewerPosts: PropTypes.func,
-        togglePostMenu: PropTypes.func,
+type PostListRowProp = {
+    listId: string,
+    previousListId?: string,
+    fullWidth?: boolean,
+    shouldHighlight?: boolean,
+    loadOlderPosts: () => any,
```

Comment:
These shouldn't return type `any`. If we don't need the return type, use `void` instead.